scripts/run_tests.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
```

Comment:
don't use unittest, tests are run with pytest

Reply:
Not sure where this change came from. Wasn't me.